lib/policies_finder_publisher.rb

```diff
@@ -67,10 +67,31 @@ def details
         document_type: "policy"
       },
       show_summaries: false,
-      facets: [],
+      facets: facets,
     }
   end
 
+  def facets
+    [
+      {
+        key: "public_timestamp",
+        short_name: "Updated",
+        name: "Published",
```

Comment:
Should this be "Last Updated" so it matches with the use of "Updated"? @bmwelby / @futurefabric